package-lock.json

```diff
@@ -3116,131 +3116,17 @@
             ],
             "dependencies": {
                 "@neon-rs/load": "^0.0.4",
-                "detect-libc": "2.0.2",
-                "libsql": "^0.3.19"
+                "detect-libc": "2.0.2"
             },
             "optionalDependencies": {
-                "@libsql/darwin-arm64": "0.4.1",
```

Comment:
Hm....not sure why this huge diff is produced...
I just run `npm install` in the root of the project which automatically fix `package-lock.json`...

Reply:
I think this is due to the fix I made for having a cyclic dependency so this makes sense